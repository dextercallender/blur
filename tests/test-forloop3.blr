int main() {
    int i;
    int j;
    /* nested for loop */
    for(i=0; i<3; i=i+1) {
	for(j=0; j<2; j=j+1) {
	    println(i);
	}
    }
}
