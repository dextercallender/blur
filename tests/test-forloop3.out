0 00 11 01 12 02 1
